metadata-load/src/main/resources/application.properties

```diff
@@ -0,0 +1,25 @@
+#
+#
+# Copyright 2018 EMBL - European Bioinformatics Institute
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#
+#JDBC properties entry for Oracle server
+OracleJDBC.url=jdbc:oracle:thin
+OracleJDBC.host=ora-vm-069.ebi.ac.uk
```

Comment:
Do not store hostnames or passwords in the properties file.

Reply:
I will remove hostname also

Reply:
done